concepticondata/concepticon.tsv

```diff
@@ -3449,3 +3449,10 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3463	RIVERBANK	The physical world	The margin or shore of a river.	Person/Thing	
 3464	SHORE OF LAKE	The physical world	The margin or shore of a lake.	Person/Thing	
 3465	SHORE OF SEA	The physical world	The margin or shore of a sea or ocean.	Person/Thing	
+3466	DANGEROUS	Emotions and values	Capable of injuring or harming someone.	Property	
+3467	HELP (AID)	Social and political relations	The act of helping someone.	Person/Thing	
```

Comment:
maybe HELP (ACTION) as gloss?

Reply:
A, sorry: not HELP (ACTION), but rather HELP (ASSISTANCE)? OUr convention is not clear here, but you'd like to have a real noun in brackets to indicate it's a noun. And assistance was the first hit on a dictionary I found.